Bottom: 0488bc5f2cce62a82ca0a9b47d804c5af12e71b3
Top:    a7cf1bbbdc8f873a6a16c1d6f43f8bfa648e4f3c
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 19:24:33 +0000

Re-enable soldermask emission


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index efe337c..8d81a8e 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1040,7 +1040,7 @@ object3d_from_soldermask_within_area (POLYAREA *area, int side)
   LayerType *layer;
   POLYAREA *pa;
 
-  return NULL;
+//  return NULL;
 
   poly_Copy0 (&info.poly, area);
   info.side = side;
